includes/Classifai/Plugin.php

```diff
@@ -37,6 +37,7 @@ public function enable() {
 		add_action( 'init', [ $this, 'i18n' ] );
 		add_action( 'admin_init', [ $this, 'init_admin_helpers' ] );
 		add_action( 'admin_init', [ $this, 'add_privacy_policy_content' ] );
+		add_action( 'admin_init', [ $this, 'maybe_migrate_to_v3' ] );
```

Comment:
Have we explored running this on the WordPress update hook, instead of on `admin_init`? Maybe it doesn't matter but we could make this only run after an update has happened (so after someone updates ClassifAI)

Reply:
Tested things out and seems to work fine. I do think it's cleaner to have this only run on the update hook but this is fine